umm-spec-lib/test/cmr/umm_spec/test/migration/version_migration.clj

```diff
@@ -1129,3 +1129,40 @@
             {:URLs ["www.contact.shoo.com"],
              :Description "Contact related url description"}]
            collection-contact-persons))))
+
+(deftest migrate-1-9-tiling-identification-systems-to-1-10
+  (let [tiling-id-systems {:TilingIdentificationSystems
+                           [{:TilingIdentificationSystemName "MISR"
+                             :Coordinate1 {:MinimumValue 1.0
+                                           :MaximumValue 10.0}
+                             :Coordinate2 {:MinimumValue 1.0
+                                           :MaximumValue 10.0}}
+                            {:TilingIdentificationSystemName "Heat Miser"
+                              :Coordinate1 {:MinimumValue 11.0
+                                            :MaximumValue 20.0}
+                              :Coordinate2 {:MinimumValue 11.0
+                                            :MaximumValue 20.0}}]}
+        result (vm/migrate-umm {} :collection "1.9" "1.10" tiling-id-systems)
+        other-result (vm/migrate-tiling-identification-systems tiling-id-systems)]
```

Comment:
1. Missing 1-10 to 1-9?
2. I don't see you changing the way version_steps is handling v1.10.  Without that, I don't think these migration tests will work.  You can call migrate-umm 1.9 1.10, but it won't call the migration method you think it will. because (compare "1.9" "1.10") >0.